ffi/util.lua

```diff
@@ -60,10 +60,27 @@ int WideCharToMultiByte(
 );
 ]]
 
+local getlibprefix = function()
+    -- Apple M1 homebrew installs libraries outside of default searchpaths,
+    -- and dyld environment variables are sip-protected on MacOS, cf. https://github.com/Homebrew/brew/issues/13481#issuecomment-1181592842
+    local libprefix = os.getenv("KO_DYLD_PREFIX")
+
+    if not libprefix then
+        local std_out = io.popen("brew --prefix", "r")
+            if std_out then
+                libprefix = std_out:read("*line")
+                std_out:close()
+            end
+    end
+
+    return libprefix;
```

Comment:
nit: trailing semi-colon

Reply:
Indeed, better add some more:
```suggestion
    return libprefix;;;;;;;;;;
```